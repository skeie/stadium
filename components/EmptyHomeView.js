// @flow

import React from 'react';
import { StyleSheet, View } from 'react-native';
import { Poppins } from '../components/StyledText';
import colors from '../constants/Colors';
const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: colors.primary,
        justifyContent: 'center',
        alignItems: 'center',
        padding: 20,
    },
});

const Homescreen = () => {
    return (
        <View style={styles.container}>
            <Poppins style={{ marginBottom: 20 }} type="header">
                Welcome to Stadiumz!
            </Poppins>
            <Poppins style={{ marginBottom: 20, textAlign: 'left', width: '100%' }}>
                • An app that helps you keep track of the football matches you have attended
            </Poppins>
            <Poppins>
                • Press on the plus icon in the down right corner to starting adding photos of
                football stadiums.
            </Poppins>
        </View>
    );
};

export default Homescreen;
