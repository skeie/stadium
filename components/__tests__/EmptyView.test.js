import 'react-native';
import React from 'react';
import EmptyHomeView from '../EmptyHomeView';
import renderer from 'react-test-renderer';

it('EmptyHomeView renders correctly', () => {
    const tree = renderer.create(<EmptyHomeView />).toJSON();
    expect(tree).toMatchSnapshot();
});

