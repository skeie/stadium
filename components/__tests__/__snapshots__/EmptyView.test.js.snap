// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyHomeView renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ADF1D2",
      "flex": 1,
      "height": "100%",
      "justifyContent": "center",
      "padding": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#553555",
          "fontFamily": "poppins",
        },
        Object {
          "color": "#553555",
          "fontFamily": "poppins",
          "fontSize": 20,
        },
        Object {
          "marginBottom": 20,
        },
      ]
    }
    type="header"
  >
    Welcome to Stadiumz!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#553555",
          "fontFamily": "poppins",
        },
        undefined,
        Object {
          "marginBottom": 20,
          "textAlign": "left",
          "width": "100%",
        },
      ]
    }
  >
    • An app that helps you keep track of the football matches you have attended
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#553555",
          "fontFamily": "poppins",
        },
        undefined,
        undefined,
      ]
    }
  >
    • Press on the plus icon in the down right corner to start and select a photo of football stadiums and see the magic happen
  </Text>
</View>
`;
